core/modules/installer/js/select_releases.js

```diff
@@ -0,0 +1,12 @@
+(function ($) {
+  $(document).ready(function() {
```

Comment:
This should be written as a `Backdrop.behavior` rather than using `$(document).ready()`.